Lib/Search/SearchManager.php

```diff
@@ -278,12 +278,15 @@ public function remove($objects)
      *
      * @param mixed $object
      * @param bool $refresh
+     * @param bool $forceVersion
      *
      * @throws Exception\DoctrineSearchException
      */
-    public function flush($object = null, $refresh = false)
+    public function flush($object = null, $refresh = false, $forceVersion = false)
     {
+        $this->client->setForceVersion($forceVersion);
```

Comment:
I prefer wrapping this and the related line 289 inside `if ($forceVersion)` so we don't need to keep setting it false over and over.